queries.sql

```diff
@@ -1,3 +1,108 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered = TRUE;
+
+SELECT * FROM animals WHERE name <> 'Gabumon';
+
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL OR species = 'null';
+COMMIT;
+
+BEGIN;
+DELETE FROM animals
+WHERE date_of_birth > '2022-01-01';
+
+BEGIN;
+SAVEPOINT my_savepoint;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+  BEGIN;
+    DELETE FROM animals WHERE  date_of_birth > '2022/01/01' ;
+    SAVEPOINT SP1;
+    UPDATE animals SET weight_kg = weight_kg * -1; 
+    ROLLBACK TO SP1;
+    UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0; 
+    COMMIT;
+
+SELECT COUNT(*) AS animal_count
+FROM animals;
+
+SELECT COUNT(*) AS animal_count
```

Comment:
- [ ] I'm not able to see the right question and its answer in the queries file. Please fix that

EX:
```
Who was the last animal seen by William Tatcher?

  SELECT animals.name, vistis.visit_date
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  LEFT JOIN vets ON vets.id = visits.vet_id
  WHERE vets.name = 'William Tatcher'
  ORDER BY visits.visit_date DESC
  LIMIT 1;
```

However, you have added the images with the query, and its result in the PR description. Please add those queries in this file as well.

Reply:
NONE of these queries is part of todays project